packages/components/src/components/hds/reveal/index.hbs

```diff
@@ -7,6 +7,7 @@
   <:toggle as |t|>
     <Hds::Reveal::Toggle::Button
       aria-controls={{t.contentId}}
+      aria-describedby={{@ariaDescribedBy}}
```

Comment:
[todo] I think you want to forward the `@ariaDescribedBy` argument (otherwise the `aria-describedby` attribute is spread on the child button via the `...attributes`, but you already have it there as explicit HTML attribute)
```suggestion
      @ariaDescribedBy={{@ariaDescribedBy}}
```

Reply:
The way I have it now renders as I expect in the DOM. Can you double check that to see if I am mistaken?

Reply:
Yes, I am pretty sure. With this code:
<img width="727" height="327" alt="modified" src="https://github.com/user-attachments/assets/45569677-f5c7-42cd-b793-f203aee22723" />
this is what is rendered
<img width="1015" height="525" alt="modified" src="https://github.com/user-attachments/assets/1fe222fc-37ca-4b1e-b1ce-80772961cd44" />
and that's because of the `...attributes` operator.

This means that 1) or you forward the attribute or 2) much simpler you pass it as HTML attribute, which makes redundant the assignment in the `reveal/toggle/button.hbs` file (see comment below)

Reply:
+1, I think it would be simpler to remove the `ariaDescribedby` argument on Hds::Reveal::Toggle::Button and just rely on the HTML attribute.